.babelrc

```diff
@@ -2,7 +2,7 @@
   "presets": [
     ["env", {
       "targets": {
-        "node": "4.5"
+        "node": 6
```

Comment:
Hmmm... I'm unsure how this helps resolve the issue... Also, I seem to remember seeing log messages from babel stating that this needs to be a string.